spec/unit/dcv/configurators/carnegie_blacklight_configurator_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe Dcv::Configurators::CarnegieBlacklightConfigurator do
+  let(:config) do
+    Blacklight::Configuration.new
+  end
+  let(:configurator) { Class.new { include Dcv::Configurators::BaseBlacklightConfigurator }.new }
+
+  describe 'fulltext search configuration' do
+    context 'flag is default' do
+      before do
+        described_class.configure(config)
+      end
+
+      it 'only includes the field values in facet.field' do
+        expect(config.search_fields['fulltext_tesim']).to be_a Blacklight::Configuration::SearchField
+      end
+    end
+    context 'flag is true' do
+      before do
+        described_class.configure(config, fulltext: true)
+      end
+
+      it 'only includes the field values in facet.field' do
+        expect(config.search_fields['fulltext_tesim']).to be_a Blacklight::Configuration::SearchField
+      end
+    end
+    context 'flag is false, as per centennial site' do
+      before do
+        described_class.configure(config, fulltext: false)
+      end
+
+      it 'only includes the field values in facet.field' do
```

Comment:
did you intend for this test to also say that it 'only includes the field values in facet.field' or was that a copy error?

Reply:
Aargh! I'll fix this in a separate commit.